plugins/ScheduledReports/lang/en.json

```diff
@@ -59,6 +58,11 @@
         "EvolutionGraphsShowForPreviousN": "Evolution graphs show the evolution over the previous N %s",
         "ReportPeriod": "Report Period",
         "ReportPeriodHelp": "The period of data covered by this report. By default this is the same as the email schedule, so if the report is sent weekly, it will contain information regarding the last week.",
-        "ReportPeriodHelp2": "You can change this, however, if you want to see different information and still retain the email schedule. For example, if the email schedule is weekly, and the report period is 'day', you will get information for the last day, every week."
+        "ScheduleReportPeriodHelp": "The period of data covered by this report. By default this is the same as the report schedule, so if the report is sent weekly, it will contain information regarding the last week.",
+        "ReportPeriodHelp2": "You can change this, however, if you want to see different information and still retain the email schedule. For example, if the email schedule is weekly, and the report period is 'day', you will get information for the last day, every week.",
```

Comment:
Is this only shown for email reports, or should it changed to `scheduled reports` here as well?

Reply:
Forgot to remove this.